### PR TITLE
Submission Revision Stage: Add name and remove submission options to request form

### LINKED ISSUE #940: Venue Request: add 'name' and 'remove submission options' to the Submission Revision Stage
In order to allow the PCs to enable 'Supplementary Material' or 'Archival Status' edition after the submission deadline, we could add these two parameters to the Submission Revision Stage:

- name: parametrize the invitation name, default value should be 'Revision'
- remove_submission_options: compute this dropdown option list based on the content of the submission invitation. Update this value in every revision post process function of the VenueRequest.

### PR DESCRIPTION
**Pull request by celestemartinez**
Add option to name revision stage as well as adds all submission fields as options to remove from revision invitation. 

Fixes #940

### CODE REVIEW

File: openreview/venue_request/venue_request.py [hunk 1]
```diff
@@ -330,16 +330,21 @@ def setup_meta_review_stage(self):
     def setup_submission_revision_stage(self):
 
         submission_revision_stage_content = {
+            'submission_revision_name': {
+                'description': 'What should be the name of the submission revision button (e.g. Revision, Supplementary Material, Post-Decision Revision)? Leave empty for default option - Revision',
+                'value-regex': '.*',
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:335]: can you add `default: 'Revision`, I will pre-populate the note editor with the 'Revision' string.

File: openreview/venue_request/venue_request.py [hunk 2]
```diff
@@ -349,17 +354,17 @@ def setup_submission_revision_stage(self):
...
+                'order': 39,
                 'value-dict': {},
                 'description': 'Configure additional options in the revision. Valid JSON expected.'
             },
             'submission_revision_remove_options': {
-                'order': 39,
-                'values-dropdown': ['keywords', 'pdf', 'TL;DR'],
-                'description': 'Fields that should not be available during revision: keywords, pdf, TL;DR'
+                'order': 40,
+                'values-dropdown': ['title','authorids', 'abstract','keywords', 'pdf', 'TL;DR'],
```
Comment (melisabok) [inline openreview/venue_request/venue_request.py:366]: what about `authors`? should we add it to the list?

File: tests/test_venue_request.py [hunk 3]
```diff
@@ -903,7 +903,7 @@ def test_post_decision_stage(self, client, test_client, selenium, request_page,
                 'submission_revision_start_date': start_date.strftime('%Y/%m/%d'),
                 'submission_revision_deadline': due_date.strftime('%Y/%m/%d'),
                 'accepted_submissions_only': 'Enable revision for all submissions',
-                'submission_revision_remove_options': ['keywords', 'pdf']
+                'submission_revision_remove_options': ['keywords']
```
Comment (melisabok) [inline tests/test_venue_request.py:906]: Can you add to this test a Submission Revision Stage reference where the revision name is Supplementary_Material and it only has the supplementary_material field? then you can check if the invitations were correctly created.

File: openreview/venue_request/venue_request.py (hunk 2 above)
Comment (celestemartinez) [reply openreview/venue_request/venue_request.py:366]: Well the authors field is hidden in the submission form, right?

Comment (melisabok) [reply openreview/venue_request/venue_request.py:366]: The UI hides it but I'm not sure if the backend will try to validate it.

Comment (celestemartinez) [reply openreview/venue_request/venue_request.py]: Yep, I removed it

**Review by melisabok — APPROVED** [review]

**Comment by melisabok** [comment]
It looks good! can you add a new questions in the FAQ with instructions for: How to keep editing the submission after the deadline?

**Pull request merged**